geomstats/_backend/autograd/__init__.py

```diff
@@ -136,6 +135,15 @@
 power = _box_binary_scalar(target=_np.power)
 
 
+def squeeze(x, axis=None):
+    if axis is None:
+        return _np.squeeze(x)
+    x_shape = x.shape
+    if x_shape[axis] != 1:
```

Comment:
Do we need to create the variable x_shape? Why not `x.shape[axis] != 1`?

Reply:
Good eye, that was a leftover. Done.